tensorflow/stream_executor/dso_loader.cc

```diff
@@ -33,54 +33,63 @@ limitations under the License.
 #include "tensorflow/stream_executor/platform/logging.h"
 #include "tensorflow/stream_executor/platform/port.h"
 #include "tensorflow/stream_executor/lib/str_util.h"
+#include "external/local_config_cuda/cuda/cuda_config.h"
 
 namespace perftools {
 namespace gputools {
 namespace internal {
 
-// TensorFlow OSS configure uses the following lines to configure versions. For
-// any modifications of the format, please make sure the script still works.
-string GetCudaVersion() { return ""; }
-string GetCudnnVersion() { return ""; }
-
 /* static */ port::Status DsoLoader::GetCublasDsoHandle(void** dso_handle) {
-  return GetDsoHandle(FindDsoPath(tensorflow::internal::FormatLibraryFileName("cublas", GetCudaVersion()),
-                                  GetCudaLibraryDirPath()),
-                      dso_handle);
+  return GetDsoHandle(
```

Comment:
Made some style fixes since I have changes in this file anyway. I can revert these changes if need be.